src/Teams/ViewComponents/MarkdownScriptFormViewComponent.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Teams.ViewComponents
+{
+    public class MarkdownScriptFormViewComponent : ViewComponent
+    {
+
+        public async Task<IViewComponentResult> InvokeAsync(string questId)
```

Comment:
The name questId is confusing. Quest is a separate word in English. it's better to use "question" instead of "quest".